src/component/elements/Slider.jsx

```diff
@@ -0,0 +1,151 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/react';
+import {
+  useState,
+  Children,
+  useMemo,
+  useEffect,
+  useCallback,
+  memo,
+} from 'react';
+import { FaAngleLeft } from 'react-icons/fa';
+import { useMeasure } from 'react-use';
+
+const Arrow = ({ direction, onClick }) => (
+  <div
+    onClick={onClick}
+    css={css`
+      display: flex;
+      position: absolute;
+      top: calc(50% - 18px);
+      ${direction === 'right' ? `right: 25px` : `left: 25px`};
+      height: 36px;
+      width: 36px;
+      justify-content: center;
+      background: white;
+      border-radius: 50%;
+      cursor: pointer;
+      align-items: center;
+      border: none;
+      transition: transform ease-in 0.1s;
+      &:hover {
+        transform: scale(1.2);
+        background-color: #607d8b;
+        color: white;
+      }
+      img {
+        transform: translateX(${direction === 'left' ? '-2' : '2'}px);
+        &:focus {
+          outline: 0;
+        }
+      }
+    `}
+  >
+    <FaAngleLeft
+      style={{ transform: `scaleX(${direction === 'right' ? '-1' : '1'})` }}
+    />
+  </div>
+);
+
+const transition = 0.45;
```

Comment:
In the final result we don't see any animation which is ok. 
You should remove everything related to animations and transitions.
No need to use absolute positioning and CSS transforms.

Reply:
the animation gives some beauty to the component, this is why I use it but the final decision, in the end, is up to you, if you need me to remove all the animation then I will remove it